float32 x
float32 y
float32 z
string frame_id
bool frame_locked
---
bool success
string message
